app/src/main/java/com/claraberriel/mvpparking/mvp/view/ParkingSizeView.java

```diff
@@ -0,0 +1,47 @@
+package com.claraberriel.mvpparking.mvp.view;
+
+import android.app.Activity;
+import android.widget.EditText;
+import android.widget.Toast;
+
+
+import com.claraberriel.mvpparking.R;
+import com.claraberriel.mvpparking.databinding.ActivityMainBinding;
+
+import java.util.Objects;
+
+public class ParkingSizeView extends ParkingActivityView {
+    private final ActivityMainBinding binding;
+
+    public ParkingSizeView(Activity activity, ActivityMainBinding binding) {
+        super(activity);
+        this.binding = binding;
+    }
+
+    // input to int
+    public int getEdittextNumber() throws NumberFormatException {
+        EditText edittextNumber = binding.edittextMainNumber;
+        return Integer.parseInt(edittextNumber.getText().toString());
+    }
+
+    //reusable
+    public void showToast(String msgName) {
```

Comment:
This method can be private. Avoid abbreviations if possible, rename `msgName` to `message`.